source/templates/actions.md

```diff
@@ -255,3 +255,41 @@ will trigger the action, but the user will remain on the current page.
 
 With `preventDefault=false`, if the user clicked on the link, Ember.js
 will trigger the action *and* the user will be directed to the new page.
+
+### Modifying the action's first parameter
+
+If a `value` option for the `{{action}}` helper is specified, its value will be
+considered a property path that will be read off of the first parameter of the
+action. This comes very handy with event listeners and enables to work with
+one-way bindings.
+
+```handlebars
+<label>What's your favorite band?</label>
+<input type="text" value={{favoriteBand}} onblur={{action "bandDidChange"}}/>
```

Comment:
This is `<input>` with attribute bindings **NOT** `{{input}}`.  Attribute bindings are one way (`{{input}}` gets around this by using the `change` event and updating the internal value).